lightning.install

```diff
@@ -457,7 +458,8 @@ function lightning_update_8001() {
   // Install the body field on the basic block_content type.
   $field = FieldConfig::load('block_content.basic.body');
   if (empty($field)) {
-    lightning_core_create_config('field_config', 'block_content.basic.body', 'lightning');
+    $config = Config::forModule('lightning');
```

Comment:
This is a bit of gardening, unrelated to the new functionality. lightning_core_create_config() has been gone for a long time now, so this code was fatally broken.